src/mad_spark_alt/core/enhanced_orchestrator.py

```
@@ -12,10 +12,10 @@
 from datetime import datetime, timezone
 from typing import Any, Dict, List, Optional
 
-from .smart_orchestrator import SmartQADIOrchestrator, SmartQADICycleResult
-from .smart_registry import SmartAgentRegistry
-from .answer_extractor import EnhancedAnswerExtractor, AnswerExtractionResult
+from .answer_extractor import AnswerExtractionResult, EnhancedAnswerExtractor
 from .interfaces import GeneratedIdea, ThinkingMethod
```

Ruff (0.11.9)

`.interfaces.ThinkingMethod` imported but unused

Remove unused import: `.interfaces.ThinkingMethod`

(F401)